Sources/Scyther/Extensions/UIImageView+Extension.swift

```diff
@@ -0,0 +1,33 @@
+//
+//  UIImageView+Extension.swift
+// 
+//
+//  Created by Stefan Haider on 01.06.21.
+//
+
+#if !os(macOS)
+import UIKit
+
+extension UIImageView {
+    
+    /**
+     Trys to load an Image from an URL and sets it as image
```

Comment:
Typo on the word `tries`

Reply:
thanks for the english lesson and greetings from austria to australia ;)